### PR TITLE
[On Hold] New package: pitivi-0.95

### PR DESCRIPTION
**Pull request by the-maldridge**
Void doesn't seem to have a capable video editor at the moment, pitivi has a pleasing graphical interface and works acceptably well.  Currently it cannot find the python headers during a cross compile, any input on this would be appreciated.

### CODE REVIEW

**Comment by stpx** [comment]
@the-maldridge
Try adding: `CPPFLAGS="-I${XBPS_CROSS_BASE}/usr/include/python3.4m"`

**Comment by the-maldridge** [comment]
Nope, it still can't find the headers for Python3.4.  I suspect there is something special going on with the configure script that is causing it to fail on arm.

**Comment by stpx** [comment]
Sorry, I meant just append `CPPFLAGS="-I${XBPS_CROSS_BASE}/usr/include/python3.4m"`.

```diff
--- a/srcpkgs/pitivi/template
+++ b/srcpkgs/pitivi/template
@@ -11,7 +11,4 @@ license="LGPL-2.1"
 homepage="http://pitivi.org/"
 distfiles="https://download.gnome.org/sources/pitivi/${version}/pitivi-${version}.tar.xz"
 checksum=5f15220bcc3d02cef4edacaa19da284014caf725e41fc9cbec70469582e1d313
-
-do_build() {
-        make CPPFLAGS="-I${XBPS_CROSS_BASE}/usr/include/python3.4m"
-}
\ No newline at end of file
+CPPFLAGS="-I${XBPS_CROSS_BASE}/usr/include/python3.4m"
```

**Comment by the-maldridge** [comment]
According to the pitivi people, this is an issue with Void's python3-cairo package, is it possible this package is not functioning correctly on the arm musl builds?

**Comment by Vaelatern** [comment]
Link with issue #1063

**Comment by the-maldridge** [comment]
Given recent changes to pycairo, I believe this package is now ready for review.

**Comment by Vaelatern** [comment]
Discussing with @the-maldridge off-list to get more of the dependencies listed in the template before merging.

File: srcpkgs/pitivi/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'pitivi'
+pkgname=pitivi
+version=0.95
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config libtool intltool itstool python3.4-devel"
+makedepends="python3.4-cairo-devel gtk+3-devel gst-plugins-base1-devel"
+depends="python3.4-gobject python3.4-cairo gst-python"
```
Comment (Vaelatern) [inline srcpkgs/pitivi/template]: Seems to depend as well on `gst-editing-services`.

**Comment by PythonJedi** [comment]
Was able to achieve successful package and install, but the installed executable fails with

`Could not import 'Gtk'. Make sure you have it available.`

pygtk is installed (pkgver pygtk-2.24.0_24) which is claimed to be up to date.

**Comment by the-maldridge** [comment]
Upstream is working through some large scale refactoring.  This PR is largely out of date with what they are planning to eventually release.

**Pull request closed without merging**